ReactWindows/ReactNative/Views/ViewManagersPropertyCache.cs

```diff
@@ -8,11 +8,6 @@ static class ViewManagersPropertyCache
     {
         private static readonly IReadOnlyDictionary<string, IPropertySetter> s_shadowEmpty = new Dictionary<string, IPropertySetter>();
 
-        private static readonly object[] s_viewArgsBuffer = new object[2];
```

Comment:
Not so suspicious after all, must have been left over from when the PropertySetter stuff was refactored.